roc-wrapper.go

```diff
@@ -0,0 +1,83 @@
+package roc
+
+import "fmt"
+
+func convertErr(err uint32, errstr string) error {
+	if err == 0 {
+		return nil
+	}
+	return fmt.Errorf("%s: %d", errstr, err)
+}
+
+// Init parses the `ip` and `port` and initializes the Address object
+func NewAddress(family Family, ip string, port int) (*Address, error) {
+	var a *Address
+	err := AddressInit(a, family, ip, port)
+	return a, convertErr(err, "Error when initializing an address")
+}
+
+// Family get Address family
+func (a *Address) Family() *Family {
+	return AddressFamily(a)
+}
+
+// Ip get Address ip
+func (a *Address) Ip() *Ip {
```

Comment:
`Ip() *Ip` -> `IP() string`

(Go code style suggests avoiding camel-case for acronyms)

Reply:
done